addon/mixins/admin-model-mixin.js

```diff
@@ -0,0 +1,18 @@
+import Ember from 'ember'
+import HasManyQuery from 'ember-data-has-many-query'
+
+const {
+  Mixin
+} = Ember
+const { ModelMixin } = HasManyQuery
+
+export default Mixin.create(ModelMixin, {
+  removeRelated(...args) {
+    const adapter = this.store.adapterFor(this.constructor.modelName)
+    return adapter.removeRelated(this, ...args)
+  },
+  addRelated(...args) {
+    const adapter = this.store.adapterFor(this.constructor.modelName)
```

Comment:
Is this the right store instance?  We should use the `admin.store`

The admin store attempts to resolve the adapters within the admin namespace first so they can be overridden.  I think it makes sense to use the admin store for all things in this repo.  If we want to us the jsonAPI mixins (which are awesome btw) in the consuming apps, we should likely break that out into a separate addon.